apps/sync/index.html

```diff
@@ -5,15 +5,12 @@
     <meta charset="utf-8">
     <!-- Lazy loaded shared
     <script defer src="shared/js/async_storage.js"></script>
+    <script defer src="shared/js/sync/errors.js"></script>
```

Comment:
Only the ones that come from shared so the build process can add them to the apps folder.